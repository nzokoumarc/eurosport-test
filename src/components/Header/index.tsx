import * as React from 'react';
import { AppBar, Container, Typography } from '@material-ui/core';
import { useStyles } from './styles';

const Header: React.FC<{}> = (props) => {
  const classes = useStyles(props);
  return (
    <AppBar color="primary" className={classes.root}>
      <Container maxWidth="md">
        <Typography color="inherit" variant="body1">
          EUROSPORT | TEST
        </Typography>
      </Container>
    </AppBar>
  );
};

export default Header;
