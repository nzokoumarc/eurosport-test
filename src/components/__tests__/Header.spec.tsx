import * as React from 'react';
import { shallow } from 'enzyme';

import Header from '../Header';

describe('Header', () => {
  it('renders', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.text()).toEqual('EUROSPORT | TEST');
    expect(wrapper).toMatchSnapshot();
  });
});
