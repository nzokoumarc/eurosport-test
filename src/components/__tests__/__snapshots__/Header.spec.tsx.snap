// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<WithStyles(ForwardRef(AppBar))
  className="makeStyles-root-1"
  color="primary"
>
  <WithStyles(ForwardRef(Container))
    maxWidth="md"
  >
    <WithStyles(ForwardRef(Typography))
      color="inherit"
      variant="body1"
    >
      EUROSPORT | TEST
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(Container))>
</WithStyles(ForwardRef(AppBar))>
`;
